### DISCUSSION TITLE
Code on incremental plan of materialized view upon data change

### DISCUSSION THREAD

**Discussion by hbtoo** [apache/doris#12808] [A - General / Q&A]
Hi all,

Does anyone know where the code is for generating the incremental plan for updating the materialized view when the base table receives new data? Many thanks!
想知道物化视图的增量维护的plan生成的代码在哪里，谢谢！

Best, 
Botong

**Comment by morningman** [comment]
https://lists.apache.org/thread/t3f0mf2znc3fv21j4h8q2swryocjbl20
